### DISCUSSION TITLE
strapi-plugin-rest-cache/guide/strategy/cache-keys

### DISCUSSION THREAD

**Discussion by giscus** [strapi-community/strapi-plugin-rest-cache#58] [Documentation]
# strapi-plugin-rest-cache/guide/strategy/cache-keys

Speed-up HTTP requests with LRU cache

https://strapi-community.github.io/strapi-plugin-rest-cache/guide/strategy/cache-keys.html

**Comment by hakunatomata2** [comment]
Hi,

as I understand this description, all pages (Content-Type=pages) will be invalidated as soon as I update one single page? At least that's the behaviour I'm experiencing but it's not what we need. We'd like to only update cache for the page edited. Is this somehow feasible? 

KR

**Reply by talkyyo** [reply]
I have tried to setup a custom provider for elasticache but get this error:

Error: Could not load REST Cache provider "elasticache". You may need to install a provider plugin "yarn add strapi-provider-rest-cache-elasticache".
